src/HealthChecks.DynamoDb/DynamoDbOptions.cs

```diff
@@ -7,10 +9,13 @@ namespace HealthChecks.DynamoDb
     /// </summary>
     public class DynamoDBOptions
     {
-        public string AccessKey { get; set; } = null!;
-
+        public AWSCredentials Credentials { get; set; } = null!;

```

Comment:
```suggestion
        public AWSCredentials? Credentials { get; set; }

```

Reply:
Credentials shouldn't be nullable, as the intention is that you should always be using it (similarly to how the access key and secret worked before).

Reply:
I asked about this before https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/pull/653#discussion_r851676222

Also I saw this https://github.com/Xabaril/AspNetCore.Diagnostics.HealthChecks/issues/482#issuecomment-678003794